packages/clients/src/aptos/wallet.ts

```diff
@@ -329,8 +329,9 @@ export class ShinamiWalletSigner {
   }
 
   /**
-   * Retrieves the wallet address.
-   * @param autoCreate Whether to automatically create the wallet (off chain) if it doesn't exist yet.
+   * Retrieves the wallet address if created in Shinami.
+   * You cannot use `getAddress` to initialize a wallet address on chain. Use `initializeWalletOnChain` instead.
```

Comment:
What does this mean while it still accepts the `onChain` arg?

Reply:
Good catch. Reading through the code now I realize we're missing a case here in the signer to handle addresses created in shinami but not initialized on chain. The signer needs a way to initialize them on chain. I'll add an extra state to track if the wallet address is on chain yet and modify `getAddress` to also initialize if `onChain` is set to true on this call.